src/app/about/page.tsx

```diff
@@ -23,8 +24,26 @@ export const metadata: Metadata = {
 export default async function About() {
   const teamMemberData = await getTeamMemberData();
 
+  const teamMemberJson = teamMemberData.map((teamMember: TeamMember) => ({
+    '@context': 'https://schema.org',
+    '@type': 'Person',
+    name: teamMember.name,
+    jobTitle: teamMember.job_title,
+    description: teamMember.bio || '',
+    url: 'https://www.codeforbc.ca/about',
+    image: teamMember.profile_image_link
+      ? `https://www.codeforbc.ca${teamMember.profile_image_link}`
+      : '',
+    sameAs: [teamMember.linkedin_link, teamMember.github_link || ''],
+  }));
+
   return (
     <Container maxWidth="lg" className="about-page">
+      <Script
+        id="jsonLD_team-member"
+        type="application/ld+json"
+        dangerouslySetInnerHTML={{ __html: JSON.stringify(teamMemberJson) }}
+      />
```

Comment:
```suggestion
      <Script id="jsonLD_team-member" type="application/ld+json">
        {JSON.stringify(teamMemberJson)}
      </Script>
```

Even though it's static data, just to minimize any risk of Cross-Site Scripting, I think we should avoid `dangerouslySetInnerHTML`

My suggestion is based on the NextJS documentation here:

https://nextjs.org/docs/pages/building-your-application/optimizing/scripts#inline-scripts